libs/utils/platforms/hikey960_energy.py

```diff
@@ -0,0 +1,124 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2016, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from energy_model import (ActiveState, EnergyModelNode, EnergyModelRoot,
+                          PowerDomain, EnergyModel)
+
+from collections import OrderedDict
+
+
+#
+# the raw hikey960 energy model data is simply calculated based on Pixel OPP
+#
+
+#
+# LITTLE CA53 Core
+#
+
+CA53_cluster_active_states = OrderedDict([
+    ( 533000, ActiveState(power=4)),
+    ( 999000, ActiveState(power=16)),
+    ( 1402000, ActiveState(power=49)),
+    ( 1709000, ActiveState(power=55)),
+    ( 1844000, ActiveState(power=70))
+])
+
+CA53_cluster_idle_states = OrderedDict([
+    ('WFI', 20),
+    ('cpu-sleep', 20),
+    ('cluster-sleep', 0)
+])
+
+CA53_cpu_active_states = OrderedDict([
+    ( 533000,   ActiveState(capacity=257,   power=160)),
```

Comment:
I don't know who ever created the EAS energy model in Hikey960 board or not.
If not, all the data we have is the estimation for simulation first. 
Only after we have the hikey960 support in LISA, we can tune the following scripts for Hikey960 board.